app/models/profile_declaration.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class ProfileDeclaration < ApplicationRecord
+  delegated_type :declarable, types: %w[EarlyCareerTeacherProfileDeclaration MentorProfileDeclaration]
+  belongs_to :participant_declaration
+  belongs_to :lead_provider
+
+  scope :ect_profiles, lambda {
```

Comment:
Are there any specs for this?

Reply:
Not in this PR. Pavel was writing them and I was going to leave them out and add them in, but they're coming.